zapisy/apps/statistics/assets/components/StatisticsList.vue

```diff
@@ -99,7 +99,7 @@ export default class StatisticsList extends Vue {
           <td>{{ group.pinned }}</td>
           <td>
             <a
-              class="badge badge-sm badge-primary"
+              class="badge badge-sm bg-primary"
```

Comment:
Czy klasa `badge-sm` w ogóle istnieje w którejkolwiek wersji BS? Mam wrażenie że ta deklaracja nic tu nie robi i może nigdy nic nie robiła; proszę to potwierdzić i usunąć.
```suggestion
              class="badge bg-primary"
```